src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck("Katarzyna");
+        Machine bulldozer = new Bulldozer("Eduardo");
+        Machine excavator = new Excavator("Maks");
```

Comment:
As Mateusz mentioned in the previous comment, you should remove all constructors from Machine child classes. If Java class doesn't have any explicitly defined constructor it will use the **default** one, so in this case you will be able to create Truck/Bulldozer/Excavator instances like this:      Machine truck = new Truck();    Notice that in this case you don't have to provide "name" of the machine, because you are able to use default constructor, i.e. new Truck();    That's why you should remove "name" fields in all child classes and delete a constructors you created.